$MirOS: ports/www/dillo/v2/patches/patch-src_IO_dpi_c,v 1.1 2008/11/07 18:05:53 bsiegert Exp $
--- src/IO/dpi.c.orig	Mon Jun 22 22:02:15 2009
+++ src/IO/dpi.c	Sun Aug 23 17:03:44 2009
@@ -297,7 +297,7 @@ static int Dpi_start_dpid(void)
    pid = fork();
    if (pid == 0) {
       /* This is the child process.  Execute the command. */
-      char *path1 = dStrconcat(dGethomedir(), "/.dillo/dpid", NULL);
+      char *path1 = dStrconcat(dGethomedir(), "/.etc/dillo/dpid", NULL);
       Dpi_close_fd(st_pipe[0]);
       if (execl(path1, "dpid", NULL) == -1) {
          dFree(path1);
@@ -371,7 +371,7 @@ static char *Dpi_get_dpid_uds_dir(void)
    char buf[256], *dpid_uds_dir, *p = NULL, *nl;
 
    saved_name_filename =
-      dStrconcat(dGethomedir(), "/.dillo/dpi_socket_dir", NULL);
+      dStrconcat(dGethomedir(), "/.etc/dillo/dpi_socket_dir", NULL);
    in = fopen(saved_name_filename, "r");
    dFree(saved_name_filename);
 
